central/metrics/custom/image_vulnerabilities/tracker.go

```diff
@@ -15,28 +16,32 @@ func New(ds deploymentDS.DataStore) *tracker.TrackerBase[*finding] {
 		"image_vuln",
 		"image vulnerabilities",
 		lazyLabels,
-		func(ctx context.Context, md tracker.MetricDescriptors) iter.Seq[*finding] {
-			return trackVulnerabilityMetrics(ctx, md, ds)
+		func(ctx context.Context, _ tracker.MetricDescriptors) iter.Seq[*finding] {
+			return track(ctx, ds)
 		},
 	)
 }
 
-func trackVulnerabilityMetrics(ctx context.Context, _ tracker.MetricDescriptors, ds deploymentDS.DataStore) iter.Seq[*finding] {
+func track(ctx context.Context, ds deploymentDS.DataStore) iter.Seq[*finding] {
 	return func(yield func(*finding) bool) {
-		finding := &finding{}
-		_ = ds.WalkByQuery(ctx, search.EmptyQuery(), func(deployment *storage.Deployment) error {
-			finding.deployment = deployment
+		var f finding
```

Comment:
why movign from `finding := &finding{}`? It just makes diff bigger with no extra value. I'd prefer to keep old name and pointer.

Reply:
Or if we never return nill then maybe better return `Seq[finding]`

Reply:
I find `var f finding` to look nicer than `f := &finding{}` by some reason. I decided to bring this minor change in this small PR.

`*finding` in `Seq[*finding]` is the type of yielded values. No reason to yield copies, hence the pointer.